docs/src/understand/proteus-to-mls.md

```diff
@@ -0,0 +1,153 @@
+
+2. Migration finalisation field:
+
+`finaliseRegardlessAfter` - date string (ISO 8601 format) until the migration has to finalise - protocol field changes to "mls", doesn’t matter if all the users (clients) have joined the MLS group. If this value is not set, migration is never forced.
+
+- Example configuration:
+
+```yaml
+mlsMigration:
+  config:
+    startTime: "2023-05-16T09:25:27.123Z"
+    finaliseRegardlessAfter: "2023-05-16T09:25:27.123Z"
+  lockStatus: "locked"
+  status: "enabled"
+  ttl: "unlimited"
+```
+
+### Team Management Panel
+
+The Team Management Admin Panel in Wire is undergoing significant updates, particularly concerning the user interface (UI) adjustments for managing the "mls" and "mlsMigration" feature configurations. 
+
+These changes are designed to enhance the control and visibility for team administrators, especially regarding the new MLS protocol implementation.
+
+#### Adjustments to "mls" Feature Configuration Interface
+
+1. **Handling of "mls" Feature When 'Locked'**:
+   - When the "mls" feature is enabled and set to "locked" status, its configuration remains visible within the Team Management feature customization panel. This visibility is contingent on the activation of a corresponding feature flag.
+   - In this locked state, the "mls" configuration settings become read-only. Team administrators can view the current configuration settings, including cipher suite and default protocol settings, but cannot make any modifications. This read-only mode is visually indicated by the greying out and disabling of all configuration fields.
+   - The locked status typically reflects settings that have been predefined at the backend level and are not subject to change at the team administration level.
+
+2. **Behavior of "mls" Feature When 'Unlocked'**:
+   - Conversely, if the "lockStatus" of the "mls" feature is set to "unlocked", the UI retains its traditional functionality.
+   - In this state, team administrators retain the ability to modify and save the configuration settings for the "mls" feature. This flexibility allows for dynamic adaptation to specific team needs.
+
+#### Introduction of "mlsMigration" Feature Configuration Panel
+
+1. **New Panel for "mlsMigration" Feature**:
+   - An addition to the Team Management UI is a new configuration panel dedicated to the "mlsMigration" feature.
+   - This panel is designed to allow team administrators to set up or update the settings pertinent to the MLS migration process.
+
+2. **Modifiability and Lock Status of "mlsMigration" Feature**:
+   - The modifiability of the "mlsMigration" feature configuration follows the same principles as other features in the Team Management panel.
+   - If the "mlsMigration" feature is "locked", the configuration becomes read-only, mirroring the behavior outlined for the "mls" feature under similar circumstances. In this state, team administrators can view but not alter the settings.
```

Comment:
I don't think is the case, at least this is not what the customer requested. They wanted it to be hidden as locked features were previously.

@tlebon @PatrykBuniX  ??

Reply:
@rohan-wire the document says: 

The "mlsMigration" feature will be hidden in team management by setting the team-settings envVar FEATURE_ENABLE_MLS_MIGRATION to false. NO information about migration config on team-management side will be visible for Bund (as per customer’s request).

but that's in a "for bund" section which I think I was asked to ignore (as the doc is supposed to be "general").

see also the rest of https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/792592579/MLS+migration+feature+config